### PR TITLE
JFrog Platform Cloud

### PR DESCRIPTION
**Pull request by danielmkn**
### What does this PR do?

New tile for JFrog Platform Integration, new Dashboard.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### CODE REVIEW

File: jfrog_platform_cloud/README.md
Comment (alai97) [inline jfrog_platform_cloud/README.md:12]:
```suggestion
This dashboard provides a summary of access logs and information about various HTTP response codes, HTTP 500 errors, calls volume, and users operations.
```

File: jfrog_platform_cloud/README.md
Comment (alai97) [inline jfrog_platform_cloud/README.md:18]:
```suggestion
Contact a JFrog representative to enable the integration. Be ready to provide your [Datadog API key][4].
```

File: jfrog_platform_cloud/README.md
Comment (alai97) [inline jfrog_platform_cloud/README.md:20]:
```suggestion

## Support

Need help? Contact [JFrog Support][5].

[1]: https://jfrog.com/
```

File: jfrog_platform_cloud/README.md
Comment (alai97) [inline jfrog_platform_cloud/README.md:24]:
```suggestion
[5]: https://support.jfrog.com/
```

File: jfrog_platform_cloud/manifest.json
Comment (alai97) [inline jfrog_platform_cloud/manifest.json:48]:
```suggestion
    "homepage": "https://jfrog.com/",
```

File: jfrog_platform_cloud/manifest.json
Comment (alai97) [inline jfrog_platform_cloud/manifest.json:5]: Will want this to be `true` to publish this as a docs page.

```suggestion
  "display_on_public_website": true,
```

**Review by alai97 — COMMENTED** [review]
Looks good, some suggestions!

**Comment by danielmkn** [comment]
Thanks @alai97!

**Review by alai97 — APPROVED** [review]
Looks good!

File: jfrog_platform_cloud/.github/CODEOWNERS
Comment (emarsha94) [inline jfrog_platform_cloud/.github/CODEOWNERS:1]: You will need to add your integration name and github handle in alphabetical order within this file. Please edit the file to include the above. To edit via web ui: 

1. Go to your forked repository > .github > CODEOWNERS file
2. Edit the file by clicking the pencil icon in the top right 
3. Add your integration name and github handle in alphabetical order
4. Click Commit changes directly to the master branch

File: jfrog_platform_cloud/README.md
Comment (emarsha94) [inline jfrog_platform_cloud/README.md:5]: Could you provide more insight on the value customers will get from this integration?

File: jfrog_platform_cloud/README.md
Comment (emarsha94) [inline jfrog_platform_cloud/README.md:12]: you can include this in the overview section. For example, this integration comes with an out-of-the-box dashboard that provides insight on xyz

File: jfrog_platform_cloud/README.md
Comment (emarsha94) [inline jfrog_platform_cloud/README.md:18]: Can you link who they should be reaching out to?

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: jfrog_platform_cloud/.github/CODEOWNERS
Comment (danielmkn) [reply jfrog_platform_cloud/.github/CODEOWNERS:1]: Removed this file and added requested changes to `.github/CODEOWNERS`

File: jfrog_platform_cloud/README.md
Comment (danielmkn) [reply jfrog_platform_cloud/README.md:5]: Added.

File: jfrog_platform_cloud/README.md
Comment (danielmkn) [reply jfrog_platform_cloud/README.md:12]: Moved to the Overview section.

File: jfrog_platform_cloud/README.md
Comment (danielmkn) [reply jfrog_platform_cloud/README.md:18]: Added.

**Comment by danielmkn** [comment]
@emarsha94, please check if everything is fixed.

File: .github/CODEOWNERS
Comment (emarsha94) [inline .github/CODEOWNERS:373]: please add @DataDog/ecosystems-review to the codeowners. Thanks!

File: jfrog_platform_cloud/README.md
Comment (emarsha94) [reply jfrog_platform_cloud/README.md:5]: @danielmkn apologies for the confusion - To clarify, the overview should include: 

1. What does your product do (in 1-2 sentences)?
2. What will customers get from this integration, and why is it valuable to them?
3. What types of data will your integration send in or pull out, and what's the value of that data? ex: By sending logs to Datadog, you have visibility into xyz

File: jfrog_platform_cloud/README.md
Comment (emarsha94) [inline jfrog_platform_cloud/README.md:12]:
```suggestion
```

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS
Comment (danielmkn) [reply .github/CODEOWNERS:373]: Added for ll JFrog records in CODEOWNERS.

File: jfrog_platform_cloud/README.md
Comment (danielmkn) [reply jfrog_platform_cloud/README.md:12]: Should I just remove it completely?

File: jfrog_platform_cloud/README.md
Comment (danielmkn) [reply jfrog_platform_cloud/README.md:5]: Done.

File: jfrog_platform_cloud/README.md
Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:8]:
```suggestion
JFrog’s SaaS Log Streamer is a log streaming solution built by JFrog for SaaS customers. This solution will stream JFrog Artifactory logs from the customer’s JFrog SaaS instance straight into their Datadog instance.
```

File: jfrog_platform_cloud/README.md
Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:10]:
```suggestion
Customers who use both JFrog and Datadog will be able to visualize Artifactory logs inside pre-configured Datadog dashboards. This integration also has built-in support for Datadog log pipelines which means logs streamed from JFrog will be preprocessed and automatically converted into the Datadog log format, allowing teams to uniquely name logs per their needs, drill down into Artifactory logs through searchable facets, and easily monitor their JFrog SaaS instance.
```

File: jfrog_platform_cloud/README.md
Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:12]:
```suggestion
At the initial release this integration will stream the following artifactory logs to Datadog:
```

File: jfrog_platform_cloud/README.md
Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:18]:
```suggestion
These logs will allow customers to readily know who accessed what repositories and how often. The logs will also show what IP addresses accessed those repositories. Log types such as traffic.log, artifactory-access.log and more request logs will be added to this integration in future updates.
```

File: jfrog_platform_cloud/README.md
Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:24]: For API integrations, we require the setup to be self-service. A user should be able to go into the JFrog platform and (securely) enter their API key and complete any other setup for this integration. Is that something your team has implemented?

File: jfrog_platform_cloud/assets/dashboards/jfrog_platform_cloud_overview.json
Comment (anawishnoff) [inline jfrog_platform_cloud/assets/dashboards/jfrog_platform_cloud_overview.json:1]: This dashboard is looking good, but please also add a description (you can edit the description by clicking on the title in the top left corner of the dashboard).

File: jfrog_platform_cloud/manifest.json
Comment (anawishnoff) [inline jfrog_platform_cloud/manifest.json:16]:
```suggestion
        "caption": "JFrog Artifactory Logs dashboard - HTTP requests",
```

File: jfrog_platform_cloud/manifest.json
Comment (anawishnoff) [inline jfrog_platform_cloud/manifest.json:21]:
```suggestion
        "caption": "JFrog Artifactory Logs dashboard - Request Logs",
```

File: jfrog_platform_cloud/manifest.json
Comment (anawishnoff) [inline jfrog_platform_cloud/manifest.json:26]:
```suggestion
        "caption": "JFrog Artifactory Logs dashboard - Operations",
```

File: jfrog_platform_cloud/manifest.json
Comment (anawishnoff) [inline jfrog_platform_cloud/manifest.json:31]: Are any other categories relevant to this integration? You can find the list here: https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags

File: jfrog_platform_cloud/manifest.json
Comment (anawishnoff) [inline jfrog_platform_cloud/manifest.json:45]: You will also need an `integration` object here. See [manifest file docs](https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file) and [Twingate](https://github.com/DataDog/integrations-extras/blob/master/twingate/manifest.json) as an example.

**Review by anawishnoff — CHANGES_REQUESTED** [review]

File: jfrog_platform_cloud/README.md
Comment (danielmkn) [reply jfrog_platform_cloud/README.md:24]: Right now the log streamer integration is in beta and we have to manually add connection details to our app.
The self-service part is dev-complete, but we are in the testing stage and it's not yet released.

File: jfrog_platform_cloud/README.md
Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:8]:
```suggestion
JFrog's SaaS Log Streamer is a log streaming solution built by JFrog for SaaS customers. This solution will stream JFrog Artifactory logs from the customer’s JFrog SaaS instance straight into their Datadog instance.
```

File: jfrog_platform_cloud/README.md
Comment (anawishnoff) [reply jfrog_platform_cloud/README.md:24]: Got it. This tile won't be published until everything is tested and complete, so can you please update the README to reflect that final stage?

File: jfrog_platform_cloud/assets/dashboards/jfrog_platform_cloud_overview.json
Comment (anawishnoff) [inline jfrog_platform_cloud/assets/dashboards/jfrog_platform_cloud_overview.json:1]: Sorry for any previous confusion - can you please add a description to this dashboard by clicking the title in the top left corner and adding the text there?

**Review by anawishnoff — CHANGES_REQUESTED** [review]
Adding a few more comments, and there are also a few of mine unaddressed from last review. Let me know if there are any questions. Thanks!

File: jfrog_platform_cloud/README.md
Comment (danielmkn) [reply jfrog_platform_cloud/README.md:24]: Updated, added steps for self-service.

File: jfrog_platform_cloud/manifest.json
Comment (danielmkn) [reply jfrog_platform_cloud/manifest.json:45]: Added.

File: jfrog_platform_cloud/manifest.json
Comment (danielmkn) [reply jfrog_platform_cloud/manifest.json:31]: Modified to better reflect the integration purpose.

File: jfrog_platform_cloud/assets/dashboards/jfrog_platform_cloud_overview.json
Comment (danielmkn) [reply jfrog_platform_cloud/assets/dashboards/jfrog_platform_cloud_overview.json:1]: Added.

File: jfrog_platform_cloud/README.md
Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:24]:
```suggestion
On the [MyJFrog Portal](https://my.jfrog.com), go to Settings -> JFrog Cloud Log Streaming - BETA, and enable the Log Streamer.
```

File: jfrog_platform_cloud/README.md
Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:25]: I would add a first step at the beginning of this Installation section for the user to create an API key inside of Datadog, and save it to a file.

Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:25]:
```suggestion
Select Datadog as the vendor. Select the file with your Datadog API key, add the Datadog intake URL for your [Datadog site](https://docs.datadoghq.com/getting_started/site/) (`https://http-intake.logs.datadoghq.com/api/v2/logs`), `ddtags` and click Save.
```

File: jfrog_platform_cloud/README.md
Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:26]:
```suggestion
Your logs will start streaming into Datadog in 24 hours or less.
```

**Review by anawishnoff — APPROVED** [review]
Looking great - just a few last comments from me!

File: jfrog_platform_cloud/README.md
Comment (danielmkn) [reply jfrog_platform_cloud/README.md:25]: Added.

Comment (danielmkn) [reply jfrog_platform_cloud/README.md:25]: Added.

File: jfrog_platform_cloud/README.md
Comment (emarsha94) [inline jfrog_platform_cloud/README.md:30]: As mentioned in the meeting this week, please update the instructions. Once done, this looks good to approve!

File: jfrog_platform_cloud/manifest.json
Comment (nmuesch) [inline jfrog_platform_cloud/manifest.json:46]:
```suggestion
```

You can remove these keys if there are no assets of that type (i.e. no recommended monitors)

File: jfrog_platform_cloud/manifest.json
Comment (nmuesch) [inline jfrog_platform_cloud/manifest.json:57]: You'll need to add a "metrics" section like this - https://github.com/DataDog/integrations-extras/blob/master/flagsmith/manifest.json#L38-L42 or remove the `metadata.csv` file if your integration doesn't emit metrics.

File: jfrog_platform_cloud/manifest.json
Comment (danielmkn) [reply jfrog_platform_cloud/manifest.json:46]: Removed.

File: jfrog_platform_cloud/manifest.json
Comment (danielmkn) [reply jfrog_platform_cloud/manifest.json:57]: `metadata.csv` is removed.

File: jfrog_platform_cloud/README.md
Comment (danielmkn) [reply jfrog_platform_cloud/README.md:30]: Updated, now users can paste the key in plain text.

File: jfrog_platform_cloud/README.md
Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:26]: Hey @danielmkn I thought this step was going to be changed so that the user did not have to encrypt/save their API key to a file?

Comment (anawishnoff) [inline jfrog_platform_cloud/README.md:26]:
```suggestion
Create [Datadog API key](https://app.datadoghq.com/organization-settings/api-keys), encrypt it with JFrog provided GPG key and save it to a text file.  
```

File: jfrog_platform_cloud/manifest.json
Comment (anawishnoff) [inline jfrog_platform_cloud/manifest.json:34]: Is this supported on any other OSes?

Comment (danielmkn) [reply jfrog_platform_cloud/manifest.json:34]: I'm a bit confused about the scope of the question. Artifactory Cloud is only Linux, but for the customers, it doesn't make any difference. But the integration itself will work on any OS, that can run DataDog web app.

Comment (danielmkn) [reply jfrog_platform_cloud/manifest.json:34]: Added.

File: jfrog_platform_cloud/README.md
Comment (danielmkn) [reply jfrog_platform_cloud/README.md]: Oh, sorry, missed that one. Fixed now.

**Comment by anawishnoff** [comment]
@danielmkn Changing that macOS line should hopefully resolve the failing validations

File: jfrog_platform_cloud/manifest.json
Comment (danielmkn) [reply jfrog_platform_cloud/manifest.json]: Changed.

**Review by bgoldberg122 — CHANGES_REQUESTED** [review]

**Review by steveny91 — APPROVED** [review]
Looks good on the integration side.

File: jfrog_platform_cloud/images/jfrog-platform-cloud.png
Comment (danielmkn) [reply jfrog_platform_cloud/images/jfrog-platform-cloud.png:1]: Removed.

File: jfrog_platform_cloud/manifest.json
Comment (FlorentClarret) [inline jfrog_platform_cloud/manifest.json:44]: Is the `\n` in the name of this dashboard intended?

**Review by FlorentClarret — APPROVED** [review]

File: jfrog_platform_cloud/manifest.json
Comment (danielmkn) [reply jfrog_platform_cloud/manifest.json]: Thanks for catching this. It's not, removed.

**Pull request merged**
